### ISSUE TITLE
alt text of images not visible in dark theme (a11y)

### ISSUE THREAD

**Issue by glmvc** [mdn/content#27414]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Performance/Animation_performance_and_frame_rate

### What specific section or headline is this issue about?

it's about the styling of img elements in general

### What information was incorrect, unhelpful, or incomplete?

I'm not quite sure where this issue belongs, but I've already mentioned it in #27241 (it was said to file a separate issue)

The styling of `img` elements sets a white background (index.scss).
`.main-page-content img { background: #fff; }`

The problem is that the alt text of images is not visible due to the background color set (see screenshots below).

I suggest to set the background-color to `transparent` for the dark theme (or `unset` it), or to set the text `color` to a (valid contrast ratio) dark color (e.g. the primary text color in light mode - but not with the custom property, because the custom property is also changed in the dark theme...).

### What did you expect to see?

The alt text to be readable using the dark theme

### Do you have anything more you want to share?

**light theme**

![light-theme-img](https://github.com/mdn/content/assets/95487764/ea3df43e-bfce-4792-affd-3ad777a85e37)

---

**dark theme**

![dark-theme-img](https://github.com/mdn/content/assets/95487764/b5efe1ce-6c13-4c9a-8c54-87f70fdfd76c)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/performance/animation_performance_and_frame_rate`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Performance/Animation_performance_and_frame_rate
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/performance/animation_performance_and_frame_rate/index.md
* Last commit: https://github.com/mdn/content/commit/6bff95b8adb7ecb18660320721433525a26360d1
* Document last modified: 2023-02-24T06:17:52.000Z

</details>

**Comment by CBID2** [comment]
Hey @glmvc! :) I setup the PR. I can't find the corresponding CSS file so can you link it here, so that I can make the necessary changes please?

**Comment by glmvc** [comment]
Hi @CBID2 👋 
Thank you for your interest in fixing this issue!
After a little search I found the mentioned file and css rule in the [Yari repository](https://github.com/mdn/yari):
https://github.com/mdn/yari/blob/394c348862eca4fa793b00fd8512117c7a394806/client/src/document/index.scss#L108-L116

I think the issue is going to be moved there for that reason (sorry for the circumstances admins, but I wasn't sure myself...).

Also, I think a separate pull request should be created in the corresponding repo...
Your [pr](https://github.com/mdn/content/pull/27423) addresses/fixes content issues, which is fine for this repository, but styling fixes should be made in the proper repo, which in this case is [mdn/yari](https://github.com/mdn/yari).

**Comment by CBID2** [comment]
Ohh ok @glmvc! :) When you say styling, do you mean CSS?
